packages/client/utils/retroGroup/updateClonePosition.ts

```diff
@@ -24,18 +51,61 @@ const getTransition = (isClipped: boolean, timeRemaining: number) => {
     : transition
 }
 
+export const getSpotlightAnimation = (
+  element: HTMLDivElement,
+  targetId: string | null | undefined,
+  groupIdsInSpotlight: string[],
+  isClose: boolean,
+  lastZIndex: number | undefined
+) => {
+  const spotlightEl = document.getElementById('spotlight')
+  const isTargetInSpotlight = targetId && groupIdsInSpotlight.includes(targetId)
```

Comment:
let's concat the `sourceReflectionGroupId` into this array, that way if a remote drag hovers the card at the top it shows above

Reply:
Added, but it does not seem to be working. Rotating spotlight card on remote browser does not have `targetId`. Checking what is possible to do.

Reply:
@nickoferrall can you please advise? 

Recently, we removed droppable attribute from the spotlight source reflection with circular animation (#5556)
Now, because of this, hovering above the source reflection cause remote reflection to `fadeOut`, as `targetId` is not submitted.

As I see we pick `targetId` from `DragAttribute.DROPPABLE` here
https://github.com/ParabolInc/parabol/blob/7ade67ab0cf2af96878785a5c30752e634dd8def/packages/client/utils/retroGroup/getTargetGroupId.ts#L18

Maybe we need to set an additional always existing attribute like `data-target-id`?

Reply:
Ah so I actually disagree that we should show remote drags above the source. We want to show remote drags above the Spotlight when they're hovering over a result group so that if they're grouped, the remote reflection fades into the Spotlight rather than suddenly appearing.

However, remote reflections can't be dropped onto the circular animating source, so I don't think there's any advantage to showing them above the source.

Reply:
@nickoferrall thank you
@mattkrick what do you think? Should I remove this concatenation?

Reply:
sure, safe to leave this one be

Reply:
@mattkrick Do you mean keep my change as is?

Reply:
Yeah, I think he means leave it how it is, i.e. don't make any changes